### PR TITLE
Improve chapter 6 overview

### PR DESCRIPTION
**Pull request by CunliangGeng**
To solve issues in #42.

### CODE REVIEW

**Comment by CunliangGeng** [comment]
@wbaccinelli you're familiar with this chapter, so I'm asking you to review it 🍺

File: episodes/chap6_setup_node.md [hunk 1]
```diff
@@ -51,15 +51,18 @@ In this lesson, you will use your laptop, but in a production scenario, we recom
...
+- Recommended operating system: Ubuntu 20.04+ , MacOS Big Sur+, or Windows 10+
+- Docker Desktop (Windows, MacOS) or the Docker Engine (Linux)
+- Miniconda (latest version)
+- Python v3.10
+- Python packages:
+  - [jupyterlab](https://pypi.org/project/jupyterlab/)
+  - [vantage6==4.7.x](https://pypi.org/project/vantage6/)
+  - [vantage6-client==4.7.x](https://pypi.org/project/vantage6-client/)
+  - [vantage6-algorithm-tools==4.7.x](https://pypi.org/project/vantage6-algorithm-tools/)
+- A code editor [Visual Studio Code](https://code.visualstudio.com/), [PyCharm](https://www.jetbrains.com/pycharm/) or something similar
```
Comment (wbaccinelli) [inline episodes/chap6_setup_node.md:63]: Miniconda, jupyterlab and the editor are not mandatory for installing the node, right? The are needed for the lesson, but amybe we should specify this?

File: episodes/chap6_setup_node.md [hunk 2]
```diff
@@ -251,9 +246,7 @@ then it will start the node and print out the following messages:
 [warn ] - private key file provided ***/private_key.pem, but does not exists
 [info ] - Setting up databases
 [info ] -   Processing csv database default: ***/data.csv
-[debug] -   - non file-based database added
```
Comment (wbaccinelli) [inline episodes/chap6_setup_node.md:254]: I think that, if the db file has been correctly mounted, the log says "file-based database added". Should we leave the correct log line?

File: episodes/chap6_setup_node.md [hunk 3]
```diff
@@ -251,9 +246,7 @@ then it will start the node and print out the following messages:
...
 [info ] -   Processing csv database default: ***/data.csv
-[debug] -   - non file-based database added
```
Comment (wbaccinelli) [inline episodes/chap6_setup_node.md:256]: Same here.

**Review by wbaccinelli — APPROVED** [review]
Well done! I have only a few minor comments.

File: episodes/chap6_setup_node.md (hunk 1 above)
Comment (CunliangGeng) [reply episodes/chap6_setup_node.md]: Only jupyterlab is not used in this chapter, so it's removed.

File: episodes/chap6_setup_node.md (hunk 2 above)
Comment (CunliangGeng) [reply episodes/chap6_setup_node.md:254]: It's debug message, safe to remove it for end users.

**Pull request merged**